README.md

```diff
@@ -11,73 +17,100 @@ Takeaway Challenge
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+#### Dish
+* An instance of Dish class has name and price attributes that can be read by other classes, namely Menu
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
+#### Message
+* An instance of Message class is initialized by the Takeaway's complete_order method, which also calls Message's send method
+* Send will invoke the Twilio API. It also calculates the arrival time by adding 60 minutes to the current time (Time.now) and formats it correctly to 24hr clock
+* Used I've used environment variables to protect confidential data needed for Twilio API (nb: I also created a twilio.env.template as an example of my twilio.env file which is not uploaded to github)
 
-As a customer
-So that I am reassured that my order will be delivered on time
-I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+### Manual feature testing in irb | Example of a customer interaction with my Takeaway program 
 ```
+katbrooks@Kats-Air takeaway % irb -r ./lib/takeaway.rb
+ => 
+#<Takeaway:0x000000014d44ae78
+... 
+3.0.2 :002 > t.view_menu
+Curry House Menu
+dal: £6.50
+korma: £3.00
+rice: £2.50
+ => {"dal"=>6.5, "korma"=>3.0, "rice"=>2.5} #not sure how to avoid this line appearing
+3.0.2 :003 > t.add_to_order("korma")
+ => "1x korma, at £3.00 each, added to your order" 
+3.0.2 :004 > t.add_to_order("dal", 3)
+ => "3x dal, at £6.50 each, added to your order" 
+3.0.2 :005 > t.view_order
+Your order contains:
+1x korma = £3.00
+3x dal = £19.50
+Order total = £22.50
+
+ => nil #not sure how to avoid this line appearing
+3.0.2 :006 > t.complete_order
+ => "Thank you for your order, which totals to £22.50. Your food will arrive by 11:48" 
+3.0.2 :007 > exit
+
+katbrooks@Kats-Air takeaway % irb -r ./lib/menu.rb
+3.0.2 :001 > m = Menu.new
+ => #<Menu:0x00000001351f0d70 @dishes={"dal"=>6.5, "korma"=>3.0, "rice"=>2.5}> 
+3.0.2 :002 > m.add_dish("chapati", 1.50)
+ => 1.5 
+3.0.2 :003 > m.dishes
+ => {"dal"=>6.5, "korma"=>3.0, "rice"=>2.5, "chapati"=>1.5} 
+3.0.2 :004 > m.display
+Curry House Menu
+dal: £6.50
+korma: £3.00
+rice: £2.50
+chapati: £1.50
+ => {"dal"=>6.5, "korma"=>3.0, "rice"=>2.5, "chapati"=>1.5} 
+ ``` 
+
+### Problems/questions
+
+1) My main concerns were around testing. Not sure if I'm using test doubles correctly and therefore if I've got sufficient test coverage. (nb: when I run rspec, it shows 100% coverage, but when I run rspec on individual spec folders, it shows <100% coverage.)
+
+2) Is there a way to test if one class' method creates a new instance of another class? Not sure how to test this is happening correctly (e.g. testing that Takeaway's complete_order method creates Message.new)
```

Comment:
Wish I could help you with some of these mock/stub questions. Definitely ask a coach